UI/GameSettingsScreen.cpp

```diff
@@ -950,6 +950,10 @@ UI::EventReturn GameSettingsScreen::OnChangeBackground(UI::EventParams &e) {
 		if (System_GetPropertyInt(SYSPROP_HAS_IMAGE_BROWSER)) {
 			System_SendMessage("bgImage_browse", "");
 		}
+#if defined(__ANDROID__)
+        System_SendMessage("bgImage_select", 
```

Comment:
Why can't both platforms just use bgImage_browse?  If it needs the path parameter, it won't hurt for browse to take it.

-[Unknown]